Description: Change the default path of the urls.txt file
 Siege installs the urls.txt in the $sysconfdir, but it wrongly
 searches for it at $prefix/etc/. This patch fixes that problem using
 the constant URL_FILE.
Author: Josue Abarca <jmaslibre@debian.org.gt>
Bug-Debian: http://bugs.debian.org/610420
Forwarded: yes
Last-Update: 2011-06-30

--- a/src/init.c
+++ b/src/init.c
@@ -99,8 +99,10 @@
   }
   
   if(strlen(my.file) < 1){ 
-    strcpy( my.file, SIEGE_HOME );
-    strcat( my.file, "/etc/urls.txt" );
+    snprintf(
+      my.file, sizeof( my.file ),
+      "%s", URL_FILE
+    );
   }
 
   if(strlen(my.uagent) < 1) 
